### PR TITLE
Add zipkin,kafka test cases

### PR DESCRIPTION
**Pull request by sakows**

### CODE REVIEW

**Review by sehz — CHANGES_REQUESTED** [review]
remove .coverage file

File: istio/tests/cases/10_kafka_messages_spec.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+with context('Starting test'):
+    with it('Testing Kafka functionality'):
+        proc_a = mp.Process(target=check_kafka_logs,args=(self,kafka_mp_output))
+        proc_b = mp.Process(target=generate_kafka_mock_request)
+        proc_a.start()
+        proc_b.start()
+        proc_a.join()
+        proc_b.join()
+        result=kafka_mp_output.get()
+        expect(result).to(contain('MOCK_REQ_ID'))
```
Comment (sehz) [inline istio/tests/cases/10_kafka_messages_spec.py]: I was expecting to see test key with value? so something like.
```expect(result.get('user-agent','MOCK_UserAgent'))```

**Review by sehz — CHANGES_REQUESTED** [review]

File: istio/tests/cases/10_kafka_messages_spec.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+                expect(d['request_headers']['content-type']).to(equal('MOCK_CONTENT_TYPE'))
+                expect(d['request_headers']['x-b3-traceid']).to(equal('aaaaaaaaaaaaaaaa'))
+                expect(d['request_headers']['x-b3-parentspanid']).to(equal('bbbbbbbbbbbbbbbb'))
+                expect(d['request_headers']['host']).to(equal(configuration.GATEWAY_URL))
+                expect(d['request_size']).to(be_below(600))
+                expect(d['response_duration']).to(be_below(6000))
+                expect(d['response_code']).to(equal(200))
+                expect(d['response_size']).to(be_below(6000))
+            else:
+                print "Request to productpage did not found among messsages. Please, run test again."
```
Comment (sehz) [inline istio/tests/cases/10_kafka_messages_spec.py]: if it is found, it should fail the test

**Review by sehz — CHANGES_REQUESTED** [review]

**Comment by sakows** [comment]
Update

**Review by sehz — APPROVED** [review]

**Pull request merged**